lib/transpilers.py

```diff
@@ -0,0 +1,21 @@
+import abc
+
+from . import human, vectorbt
+
+
+class Transpiler():
+    @abc.abstractstaticmethod
+    def convert_to_string(cls, root_node: dict) -> str:
+        raise NotImplementedError()
```

Comment:
This is my contract for now. May add more methods later, just let me know what you want and we can talk.